docs/src/components/Layout/Header.tsx

```diff
@@ -113,6 +113,7 @@ export const Header = ({ platform, colorMode, setColorMode }) => {
         <Button
           className="docs-header-menu-button"
           onClick={() => setExpanded(!expanded)}
+          aria-label="Docs header menu button"
```

Comment:
we may eventually use variables for aria-label for i18n.

Reply:
Please use ariaLabel instead

Reply:
Done.